### PR TITLE
Check installation channel for operator

### PR DESCRIPTION
**Pull request by Koncpa**

### CODE REVIEW

File: Plans/packit-ci.fmf [hunk 1]
```diff
@@ -9,7 +9,7 @@ prepare:
      - dnf makecache
 
 environment:
-  TANG_IMAGE: "quay.io/sec-eng-special/fedora_tang_server"
```
Comment (sarroutbi) [inline Plans/packit-ci.fmf:12]: This should be unchanged. This is the Tang image (the Tang container), not the Tang operator container

**Review by sarroutbi — CHANGES_REQUESTED** [review]

File: Plans/packit-ci.fmf [hunk 2]
```diff
@@ -9,7 +9,7 @@ prepare:
...
+  TANG_IMAGE: "quay.io/sec-eng-special/nbde-tang-server-tang-operator-bundle"
```
[the remark this answers is not in the document]
Comment (Koncpa) [reply Plans/packit-ci.fmf:12]: @sarroutbi  Sure, I can keep it as it is, but is it most actual version regarding to latest changes??

**Comment by sarroutbi** [comment]
Hello Patrik. Are you planning to fix this?

**Comment by Koncpa** [comment]
> Hello Patrik. Are you planning to fix this?

Sure, I just not focused on this PR, right now, but it should be already adjusted...

**Review by sarroutbi — APPROVED** [review]
Changes LGTM

**Pull request merged**